core/src/components/range/range.tsx

```diff
@@ -665,79 +741,61 @@ Developers can dismiss this warning by removing their usage of the "legacy" prop
       }
     }
 
-    renderHiddenInput(true, el, this.name, JSON.stringify(this.getValue()), disabled);
+    let labelledBy;
```

Comment:
```suggestion
    let labelledBy: string;
```
A variable without a type will inherently be `any`.